python_src/tests/performance_manager/test_l0_gtfs_rt_events.py

```diff
@@ -0,0 +1,66 @@
+import os
+import pathlib
+
+from lamp_py.performance_manager.l0_rt_vehicle_positions import (
+    get_vp_dataframe,
+    transform_vp_datatypes,
+)
+from lamp_py.performance_manager.l0_rt_trip_updates import (
+    get_and_unwrap_tu_dataframe,
+)
+from lamp_py.performance_manager.gtfs_utils import (
+    add_missing_service_dates,
+)
+
+from ..test_resources import test_files_dir, csv_to_vp_parquet
+
```

Comment:
I think it would be good to get some more granular testing specifically on the `service_date_from_timestamp` function. 

Since it does have such a discrete unit of work, feed it timestamps for 2:55AM, 3:00AM and 3:05AM and make sure we get our expected service date back. 

Possibly also from a day that falls on a time change, just to make sure everything is behaving as we'd expect.